apps/landingPage/src/styles/global.css

```diff
@@ -0,0 +1,4 @@
+body {
+  margin: 0;
+  -webkit-font-smoothing: antialiased;
+}
```

Comment:
Also this should be replace with [styled-components](https://www.styled-components.com/docs/api#createglobalstyle)

Reply:
Sorry missed that, fixed now